datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -1490,6 +1500,38 @@ pub struct CreateExternalTable {
     pub options: HashMap<String, String>,
 }
 
+#[derive(Clone)]
+pub enum WriteOp {
+    Insert,
+    Delete,
+    Update,
+    Ctas,
```

Comment:
I'd love to know if folks are interested in CTAS - that's the one thing I can see actually plumbing through to DataFusion the executable.

Reply:
I think datafusion already supports create table as select (

```
❯ create table foo as values (1), (2);
0 rows in set. Query took 0.026 seconds.
❯ create table bar as select * from foo;
0 rows in set. Query took 0.009 seconds.
❯ select * from bar;
+---------+
| column1 |
+---------+
| 1       |
| 2       |
+---------+
```

Although maybe that is part of datafusion-cli directly